src/engine-actions/create-channels.js

```diff
@@ -15,10 +15,10 @@ const { CHANNEL_ROUNDING } = require('../constants')
  * uneconomic channels (that are too small to be worth it.)
  *
  * @private
- * @param   {Engine} engine
+ * @param   {Object} engine
```

Comment:
Is there a way for us to define a global type like `Engine`?

Reply:
Yup, I added a global typedef for this